pytorch_lightning/loops/batch/training_batch_loop.py

```diff
@@ -654,8 +654,4 @@ def _truncated_bptt_enabled(self) -> bool:
 
     def _truncated_bptt_steps(self) -> int:
         """Returns the number of tbptt steps"""
-        lightning_module = self.trainer.lightning_module
-        # Give precedence to the LightningModule as the Trainer flag will be removed in v1.5
-        if lightning_module.truncated_bptt_steps > 0:
-            return lightning_module.truncated_bptt_steps
-        return self.trainer.truncated_bptt_steps or 0
+        return self.trainer.lightning_module.truncated_bptt_steps > 0
```

Comment:
tests are failing I think because  here we should return 
```suggestion
        return self.trainer.lightning_module.truncated_bptt_steps or 0
```